docs/shared-content/upgrade/upgrade-disable-targets-cloned-instance.include.md

```diff
@@ -0,0 +1,5 @@
+### Disabling All Targets - Optional
+
+A cloned instance is what it says on the tin, a cloned instance.  Assuming you are not using polling Tentacles, all the deployments will "just work."  "Just working" is by design if the VM hosting Octopus Deploy is lost and you have to restore Octopus Deploy from a backup.  
```

Comment:
The meaning of "is what it says on the tin" and "double edge(d) sword" might be confusing to people who aren't native speakers.

Reply:
Fair enough, updated.